code_samples/api/public_php_api/src/Command/TaxonomyCommand.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Command;
+
+use Ibexa\Contracts\Core\Repository\PermissionResolver;
+use Ibexa\Contracts\Core\Repository\UserService;
+use Ibexa\Contracts\Taxonomy\Service\TaxonomyServiceInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class TaxonomyCommand extends Command
+{
+    private TaxonomyServiceInterface $taxonomyService;
+
+    private PermissionResolver $permissionResolver;
+
+    private UserService $userService;
+
+    public function __construct(TaxonomyServiceInterface $taxonomyService, PermissionResolver $permissionResolver, UserService $userService)
```

Comment:
Cleaned up code style a bit:
```suggestion
    public function __construct(
        TaxonomyServiceInterface $taxonomyService, 
        PermissionResolver $permissionResolver, 
        UserService $userService
) {
```